src/app/views/people/people.component.scss

```diff
@@ -31,9 +31,9 @@ div.d-flex{
 .fa-phone,.fa-envelope{
   color: #0949ca;
 }
-*{
-  overflow-x: hidden;
-}
+// *{
```

Comment:
Did you check the output of your changes ??
<img width="831" alt="Screenshot 2024-03-26 at 4 08 02 PM" src="https://github.com/ICWaR/icwar.github.io/assets/154324075/d1eb1453-5ad4-480e-a329-e490bdfeab49">

Reply:
On my system It still looks good, and in Mobile View as well. But in between the problem is there, This I'll fix.

![Screenshot 2024-03-26 170759](https://github.com/ICWaR/icwar.github.io/assets/138218874/40dabc5e-3a6b-4a84-831c-9057ead47ce6)

Reply:
Please fix this first. 
I had instructed earlier also that we have to test it for all types of screen sizes . Google how to check different screensizes in chrome and then change accordingly

Reply:
![Screenshot 2024-03-26 201358](https://github.com/ICWaR/icwar.github.io/assets/138218874/2ba0121b-b6b5-470f-932d-b717a152c460)


Okay, Fixed it.